### ISSUE TITLE
MEDIA_STATUS message causing NullPointerException

### ISSUE THREAD

**Issue by mhilbush** [vitalidze/chromecast-java-api-v2#113]
I'm seeing a NullPointerException upon receipt of the messages below. I see this using two Google devices (Chromecast dongle and a Google Home Mini). I'm not sure if this is relevant, but the source is a SiriusXM audio stream from the SiriusXM online player.

The chromecast-java-api-v2 version 0.11.0 is embedded in the openHAB chromecast binding.
https://github.com/openhab/openhab2-addons/tree/master/bundles/org.openhab.binding.chromecast

Let me know if I can provide any more information.

```
2019-07-30 09:32:31.237 [DEBUG] [su.litvak.chromecast.api.v2.Channel ] -  <-- {"type":"MEDIA_STATUS","mediaSessionId":0,"media":{"contentId":"","streamType":"NONE","contentType":"","metadata":{"metadataType":0,"title":"We Take Your Calls","images":[{"url":""}]}},"playbackRate":1,"playerState":"IDLE","currentTime":0,"supportedMediaCommands":0,"volume":{}}
2019-07-30 09:32:31.250 [WARN ] [su.litvak.chromecast.api.v2.Channel ] - Error while handling, caused by java.lang.NullPointerException

2019-07-30 09:45:06.163 [DEBUG] [su.litvak.chromecast.api.v2.Channel ] -  <-- {"type":"MEDIA_STATUS","mediaSessionId":0,"media":{"contentId":"","streamType":"NONE","contentType":"","metadata":{"metadataType":0,"title":"Text BRAG to 606060","images":[{"url":""}]}},"playbackRate":1,"playerState":"IDLE","currentTime":0,"supportedMediaCommands":0,"volume":{}}
2019-07-30 09:45:06.163 [WARN ] [su.litvak.chromecast.api.v2.Channel ] - Error while handling, caused by java.lang.NullPointerException
```

**Comment by vitalidze** [comment]
Such format of Media Status is something new and was unexpected. I have added support for the format along with the unit test. Fix will be included into one of the next minor releases.

Thank you for your feedback.

**Comment by mhilbush** [comment]
Thanks for the quick turnaround!

**Comment by vitalidze** [comment]
Fix was included in just released version 0.11.1, which will become available today. Please let me know if the problem persists.
